epoch.go

```diff
@@ -181,7 +181,7 @@ func (e *Epoch) init() error {
 	e.maybeAssignDefaultConfig()
 	e.initOldestNotFinalizedNotarization()
 	e.oneTimeVerifier = newOneTimeVerifier(e.Logger)
-	e.sched = NewScheduler(e.Logger)
+	e.blockVerificationScheduler = NewBlockVerificationScheduler(e.Logger, DefaultMaxPendingBlocks)
```

Comment:
Let's separate `DefaultMaxPendingBlocks` from this and call this something like `DefaultProcessingBlocks` and give it a high value like 500.